### ISSUE TITLE
Piwik integration

### ISSUE THREAD

**Issue by shahar3012** [RocketChat/Rocket.Chat#233]
Piwik is the leading open-source analytics platform.
It will be very nice to have an integration with Piwik - only need to specify a siteID (a number), the url for your own piwik tracker and to put the generated code from piwik in some place on the site.
This is for the basic intergration.

It can be a bit challanging to use all of piwik features, but we might not have to - it's your choice.

As I've already said, the basic integration is very simple and easy to implement in any website.

We should have settings for that too. It can be in some config file, I really don't care, beacuse in Rocket.Chat you only config this once, and if the url and the siteID aren't changing than we have no more work to do. So UI settings doesn't have to be a requirement.

See more on: http://piwik.org/ or https://github.com/piwik/piwik

**Comment by engelgabriel** [comment]
Depends on #222

**Comment by graywolf336** [comment]
@shahar3012 and @k0nsl Just letting you know that I have started work on this and whenever this task gets closed, it will be merged into develop branch and then the Monday after it is merged it will be in the stable branch. :+1:

**Comment by rasos** [comment]
Is PIWIK now able to also track messages, rooms and users? Even when I activate those extended analysis settings, I do not see them in the PIWIK analysis pages.
